pf/tfbridge/provider_update.go

```diff
@@ -39,6 +39,7 @@ func (p *provider) UpdateWithContext(
 ) (resource.PropertyMap, resource.Status, error) {
 	ctx = p.initLogging(ctx, p.logSink, urn)
 
+	inputSecrets := findSecretPaths(checkedInputs)
```

Comment:
QQ: Should secrets be sticky over state? Perhaps if something is secret in the state, and there's no checked input for it, and the planner reuses the value from the state as the new value, this would currently un-secret it perhaps, but should not?  That is should we be looking at `priorStateMap resource.PropertyMap` as well? (not suggesting to add to the PR). This is getting into a sketchy territory for path-based heuristics since Update can use a priorStateMap value, replace it with checkedInput value, or in some cases merge the two values.